### PR TITLE
Update doc/ca.pod to clarify description for dates

### PR DESCRIPTION
**Pull request by InfoHunter**
"Note" part is based on PR #3566

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man1/ca.pod [hunk 1]
```diff
@@ -701,6 +705,13 @@ For example if the CA certificate has:
 
 then even if a certificate is issued with CA:TRUE it will not be valid.
 
+=head1 NOTE
+
+Based on RFC 5280, since OpenSSL 1.1.1, the certificate validity period
+(specified by any of B<-startdate>, B<-enddate> and B<-days>) will be
+enconded as UTCTime if the dates are earlier than 2049 (included), and as
```
Comment (FdaSilvaYY) [inline doc/man1/ca.pod]: enconded ? ;)
than (year) 2049

Comment (InfoHunter) [inline doc/man1/ca.pod]: Errr...

**Comment by InfoHunter** [comment]
commit updated. @FdaSilvaYY

File: doc/man1/ca.pod [hunk 2]
```diff
@@ -162,12 +162,16 @@ Don't output the text form of a certificate to the output file.
 =item B<-startdate date>
 
 This allows the start date to be explicitly set. The format of the
-date is YYMMDDHHMMSSZ (the same as an ASN1 UTCTime structure).
+date is YYMMDDHHMMSSZ (the same as an ASN1 UTCTime structure), or
+YYYYMMDDHHMMSSZ (the same as an ASN1 GeneralizedTime structure). In
+both formats, seconds SS and timzone Z must be presented.
```
Comment (kaduk) [inline doc/man1/ca.pod]: I think s/presented/present/ is better.

File: doc/man1/ca.pod [hunk 3]
```diff
@@ -162,12 +162,16 @@ Don't output the text form of a certificate to the output file.
...
 
 =item B<-enddate date>
 
 This allows the expiry date to be explicitly set. The format of the
-date is YYMMDDHHMMSSZ (the same as an ASN1 UTCTime structure).
+date is YYMMDDHHMMSSZ (the same as an ASN1 UTCTime structure), or
+YYYYMMDDHHMMSSZ (the same as an ASN1 GeneralizedTime structure). In
+both formats, seconds SS and timzone Z must be presented.
```
Comment (kaduk) [inline doc/man1/ca.pod]: (ditto)

**Comment by InfoHunter** [comment]
updated: change 'presented' to 'present'

**Comment by InfoHunter** [comment]
Let'e ping @openssl for an OMC approval.

File: doc/man1/ca.pod [hunk 4]
```diff
@@ -701,6 +705,13 @@ For example if the CA certificate has:
 
 then even if a certificate is issued with CA:TRUE it will not be valid.
 
+=head1 NOTE
+
+Based on RFC 5280, since OpenSSL 1.1.1, the certificate validity period
```
Comment (richsalz) [inline doc/man1/ca.pod]: Call this HISTORY and start it as "Since OpenSSL version 1.1.1, the program follows RFC 5280 and ...."

**Comment by InfoHunter** [comment]
updated based on @richsalz 's comments.

**Comment by InfoHunter** [comment]
Delete 'version' to follow #3938

**Comment by InfoHunter** [comment]
Please re-confirm, @kaduk @richsalz

**Comment by richsalz** [comment]
reconfirm

--
Senior Architect, Akamai Technologies
Member, OpenSSL Dev Team
IM: richsalz@jabber.at Twitter: RichSalz

**Comment by InfoHunter** [comment]
looks like this could be merged now...

**Comment by richsalz** [comment]
thank you!

**Pull request closed without merging**